[TAXII 2 Feed] Remove default list of objects types (#28710)
demisto/content: Packs/FeedTAXII/ReleaseNotes/1_1_32.md

#### Integrations

##### TAXII 2 Feed

> - Updated the integration to fetch all available object types instead of a predefined list of object types in the case that *STIX Objects To Fetch* parameter is not set.

Reviewer: ```suggestion
- Updated the integration to fetch all available object types instead of a predefined list of object types when *STIX Objects To Fetch* parameter is not set.
```
